app/logic/Contacts/JSON/JSONGroup.ts

```diff
@@ -0,0 +1,65 @@
+import { Group } from "../../Abstract/Group";
+import { Person } from "../../Abstract/Person";
+import { sanitize } from "../../../../lib/util/sanitizeDatatypes";
+import { backgroundError } from "../../../frontend/Util/error";
+import { assert } from "../../util/util";
+import { appGlobal } from "../../app";
+
+export class JSONGroup {
+  static save(group: Group): any {
+    assert(group.addressbook?.dbID, "Need address book ID to save the group");
```

Comment:
```suggestion
    assert(group.addressbook?.id, "Need address book ID to save the group");
```

Reply:
Compare JSONEMail, which uses `folder.id`, not `folder.dbID`.

Reply:
Same in the other code here. Please take another look at all the IDs. The JSON code is supposed to be generic and work independently from the database. (I broke that separation in person.dbID and email.dbID, but these might be a mistake of mine.)